scripts/includes/common.sh

```diff
@@ -100,23 +105,26 @@ generate_random_passwords() {
 
 configure_st2_user () {
   # Create an SSH system user (default `stanley` user may be already created)
-  if (! id stanley 2>/dev/null); then
-    sudo useradd stanley
+  SYSTEM_USER=stanley
+  if (! id ${SYSTEM_USER} 2>/dev/null); then
+    sudo useradd ${SYSTEM_USER}
   fi
 
-  sudo mkdir -p /home/stanley/.ssh
+  SYSTEM_HOME=$(get_home ${SYSTEM_USER})
```

Comment:
Can we go with simpler `SYSTEM_HOME=$(echo ~stanley)` and avoid function and `eval`?

If not, should we include the `get_home` function in respective doc as well?
https://docs.stackstorm.com/install/deb.html#configure-ssh-and-sudo

Reply:
Yes, it actually worked this time. I have no idea why my previous tests without the `eval` had failed. I've gone ahead and removed the `get_home()` function.